tests/drivers/gpio/gpio_basic_api/testcase.yaml

```diff
@@ -4,3 +4,6 @@ tests:
     depends_on: gpio
     min_flash: 34
     filter: dt_compat_enabled("test,gpio_basic_api")
+    harness: ztest
+    harness_config:
+      fixture: gpio_loopback
```

Comment:
This would deserve rewrite of the sanitycheck.rst, as it mentions things like fixture_gpio_loop, fixture_ble_fw
I prefer the one proposed in this PR indeed (w/o the 'fixture_' prefix)